manifests/network/contrail/analytics.pp

```diff
@@ -199,7 +194,6 @@
   $auth_host                  = hiera('contrail::auth_host'),
   $auth_port                  = hiera('contrail::auth_port'),
   $auth_protocol              = hiera('contrail::auth_protocol'),
-  $auth_port_ssl              = hiera('contrail::auth_port_ssl'),
```

Comment:
yes, there is appropriate change in the tripleo heat templates (that is based on https://review.openstack.org/#/c/475048/21)